app/src/main/res/values/styles.xml

```diff
@@ -17,11 +17,107 @@
 
     <style name="MountainTheme" parent="MountainThemeBase" />
 
-    <style name="WhiteThemeBase" parent="AppTheme">
-        <item name="android:windowBackground">@color/white</item>
-        <item name="colorAccent">@color/safe_green</item>
-    </style>
+    <style name="WhiteThemeBase" parent="AppTheme" />
 
     <style name="WhiteTheme" parent="WhiteThemeBase" />
 
+    <style name="PrimaryButton">
+        <item name="android:layout_height">54dp</item>
+        <item name="android:background">@drawable/primary_button</item>
+        <item name="android:gravity">center</item>
+        <item name="android:paddingStart">16dp</item>
+        <item name="android:paddingEnd">16dp</item>
+        <item name="android:textColor">@drawable/primary_button_text</item>
+        <item name="android:textSize">16sp</item>
+        <item name="fontFamily">sans-serif-medium</item>
+        <item name="textAllCaps">false</item>
+    </style>
+
+    <style name="Text" parent="TextAppearance.AppCompat">
+        <!--
+        Only use this to set font.
+        All other attributes will overwrite default values for EVERY view
+        -->
+    </style>
+
+    <style name="TextTitle" parent="Text">
+        <item name="fontFamily">sans-serif</item>
+        <item name="android:lineSpacingExtra">4sp</item>
+        <item name="android:textColor">@color/gnosis_dark_blue</item>
+        <item name="android:textSize">20sp</item>
+    </style>
+
+    <style name="TextTitleSection" parent="Text">
+        <item name="fontFamily">sans-serif</item>
+        <item name="android:lineSpacingExtra">6sp</item>
+        <item name="android:textColor">@color/gnosis_dark_blue</item>
+        <item name="android:textSize">16sp</item>
+        <item name="android:textStyle">bold</item>
+    </style>
+
+    <style name="TextDark" parent="Text">
+        <item name="fontFamily">sans-serif-medium</item>
+        <item name="android:lineSpacingExtra">6sp</item>
+        <item name="android:textColor">@color/gnosis_dark_blue</item>
+        <item name="android:textSize">16sp</item>
+        <item name="android:textStyle">normal</item>
+    </style>
+
+    <style name="TextLight" parent="Text">
+        <item name="fontFamily">sans-serif</item>
+        <item name="android:lineSpacingExtra">6sp</item>
+        <item name="android:textColor">@color/dark_grey</item>
+        <item name="android:textSize">14sp</item>
+        <item name="android:textStyle">normal</item>
+    </style>
```

Comment:
I think I will probably add additionally a `TextButton` style in my PR that inherits from TextDark problably. I guess so that we can add the ripple effect on click too.

Reply:
Can't you use primary button style?

Reply:
I suppose I could but I am concerned about the background being set. Some situations (like the "back" or "next" button on the toolbar) may need a transparent background.